docs/examples/extensions/table-column/woocommerce-admin-table-column.php

```diff
@@ -31,6 +31,27 @@ function table_column_register_script() {
 }
 add_action( 'admin_enqueue_scripts', 'table_column_register_script' );
 
+/**
+ * When adding fields to a REST API response, we need to update the schema
+ * to reflect these changes. Not only does this let API consumers know of
+ * the new fields, it also adds them to Report Exports.
+ *
+ * @param array $properties The endpoint item schema properties.
+ * @return array Filtered schema.
+ */
+function add_product_extended_attributes_schema( $properties ) {
+	$properties['extended_info']['average_rating'] = array(
+		'type'        => 'number',
+		'readonly'    => true,
+		'context'     => array( 'view', 'edit' ),
+		'description' => 'Average product rating.',
+	);
+
+	return $properties;
+}
+
+add_filter( 'woocommerce_rest_report_products_schema', 'add_product_extended_attributes_schema' );
```

Comment:
Can you add a comment for the filter?

Reply:
Do you mean for where this filter is applied? https://github.com/woocommerce/woocommerce/blob/d67381148ac192502a9387ac00f9fb181df8dcc7/includes/abstracts/abstract-wc-rest-controller.php#L75

Reply:
Oops, yes exactly.